### PR TITLE
DXCDT-543: Refining `tf generate` output instructions

### PR DESCRIPTION
**Pull request by willvedd**
### 🔧 Changes

The `auth0 tf generate` command is complex and has a few key points of failure. It is paramount that users can be guided through the process. This PR attempts to refine the instructional outputs for clarity and correctness.

### 📝 Checklist

- [x] All new/changed/fixed functionality is covered by tests (or N/A)
- [x] I have added documentation for all new/changed functionality (or N/A)

### CODE REVIEW

File: internal/ansi/ansi.go [hunk 1]
```diff
@@ -66,6 +66,11 @@ func Italic(text string) string {
 	return color.Sprintf(color.Italic(text))
 }
 
+// URL formats URL links if the writer supports it
```
Comment (Widcket) [inline internal/ansi/ansi.go:69]:
```suggestion
// URL formats URL links if the writer supports it.
```

Comment (Widcket) [reply internal/ansi/ansi.go:69]: Screenshot 2023-09-13 at 11 34 33 AM

File: internal/cli/terraform.go [hunk 2]
```diff
@@ -166,43 +166,42 @@ func generateTerraformCmdRun(cli *cli, inputs *terraformInputs) func(cmd *cobra.
...
-					inputs.OutputDIR,
-				)
-				cli.renderer.Newline()
-				cli.renderer.Infof(
-					"After running the above command and generating the state, " +
-						"the ./terraform binary and auth0_import.tf files can be safely removed.\n",
-				)
-
+			if err != nil {
+				cli.renderer.Warnf("Terraform resource config generated successfully but error with terraform plan.\n\n")
```
Comment (Widcket) [inline internal/cli/terraform.go:180]:
```suggestion
				cli.renderer.Warnf("Terraform resource config generated successfully but there was an error with terraform plan.\n\n")
```

File: internal/cli/terraform.go [hunk 3]
```diff
@@ -166,43 +166,42 @@ func generateTerraformCmdRun(cli *cli, inputs *terraformInputs) func(cmd *cobra.
...
+				cli.renderer.Warnf("Run " + ansi.Cyan(cdInstructions+"./terraform plan") + " to troubleshoot\n\n")
+				cli.renderer.Warnf("Once the plan succeeds, run " + ansi.Cyan("./terraform apply") + " to complete import.\n\n")
```
Comment (Widcket) [inline internal/cli/terraform.go:182]:
```suggestion
				cli.renderer.Warnf("Once the plan succeeds, run " + ansi.Cyan("./terraform apply") + " to complete the import.\n\n")
```

File: internal/cli/terraform.go [hunk 4]
```diff
@@ -166,43 +166,42 @@ func generateTerraformCmdRun(cli *cli, inputs *terraformInputs) func(cmd *cobra.
...
+				cli.renderer.Infof("The terraform binary and auth0_import.tf files can be deleted.\n")
```
Comment (sergiught) [inline internal/cli/terraform.go:183]:
```suggestion
				cli.renderer.Infof("The terraform binary and auth0_import.tf files can be deleted afterwards.\n")
```

**Review by sergiught — APPROVED** [review]

**Pull request merged**